### PR TITLE
Updates to contributing file

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

**Comment by Purg** [comment]
I'm planning on updating the review and release process documents a little yet, thus the draft status.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,41 +3,181 @@
...
+
+### Code Reviews
+Pull requests are reviewed by one or more of the core SMQTK-Core maintainers
+using the GitHub tools for discussions.
+Maintainers should not merge a PR until it conforms to the requirements
+described here (e.g. coding style, release notes, etc.) and it is confirmed
+that the code has sufficient unit tests and does not break any existing unit
+tests.
+
+The [review process documentation] is described in more details.
```
Comment (joshanderson-kw) [inline CONTRIBUTING.md:165]: This sentence sounds a little awkward to me. Maybe could do something like:
"See the [review process documentation] for more details on the code review process."

Comment (Purg) [reply CONTRIBUTING.md:165]: Good catch! That sentence was likely the result of an unfinished thought. I'll get an update here in a sec.

**Review by joshanderson-kw — APPROVED** [review]
LGTM!

**Comment by Purg** [comment]
Removed extra whitespace, rebased on master. Will merge when CI is green.

**Pull request merged**